### PR TITLE
Added scss to build process

### PR DESCRIPTION
**Pull request by tladendo**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -136,10 +134,11 @@ gulp.task('wiredep', function () {
     .src(config.index)
     .pipe(wiredep(options))
     .pipe(inject(js, '', config.jsOrder))
-    .pipe(gulp.dest(config.client));
+    .pipe(gulp.dest(config.client))
+    .pipe(gulp.dest(config.temp));
 });
 
-gulp.task('inject', ['wiredep', 'styles', 'templatecache'], function () {
+gulp.task('inject', ['wiredep', 'scss', 'templatecache'], function () {
```
Comment (indytechcook) [inline gulpfile.js]: does scss need be here since it's part of wiredep?

Comment (tladendo) [inline gulpfile.js]: good point, i'll get rid of it

**Comment by vietnogi** [comment]
this should onto my branch otherwise, we will be deleting history.

File: .gitignore [hunk 2]
```diff
@@ -6,11 +6,12 @@ logs
 *.log
 
 # Build files
-.tmp
```
Comment (vietnogi) [inline .gitignore]: not a big deal but curious to why

**Comment by indytechcook** [comment]
I think this is great.  Small, focused and easy to merge.

@tladendo do we want to use the scss version of bootstrap?

File: gulpfile.js [hunk 3]
```diff
@@ -492,7 +491,7 @@ function startBrowserSync(isDev, specRunner) {
   // If build: watches the files, builds, and restarts browser-sync.
   // If dev: watches less, compiles it to css, browser-sync handles reload
   if (isDev) {
-    gulp.watch([config.less], ['styles'])
+    gulp.watch([config.scss], ['scss'])
```
Comment (vietnogi) [inline gulpfile.js]: are we watching jade files somewhere?

**Comment by indytechcook** [comment]
Related to #28

File: .gitignore (hunk 2 above)
Comment (tladendo) [inline .gitignore]: just want to make sure nothing we build is tucked away – ultimately pretty inconsequential & open to discuss more but i can see the case for looking at these files during dev (i certainly have)

File: gulpfile.js (hunk 3 above)
Comment (tladendo) [inline gulpfile.js]: jade will be a separate PR

File: gulp.config.js [hunk 4]
```diff
@@ -48,7 +49,13 @@ module.exports = function () {
       '**/*.module.js',
       '**/*.js'
     ],
-    less: client + 'styles/styles.less',
+
+    scss: client + '**/*.scss',
+    scssBuild: scssBuild,
+    compass: {
+      css: temp.replace('./', ''),
+      sass: scssBuild.replace('./', '')
```
Comment (vietnogi) [inline gulp.config.js]: Should add a comment for this odd `.replace`.
